lib/util/print-version.js

```diff
@@ -0,0 +1,6 @@
+const printVersion = () => {
+  let pjson = require('../../package.json');
```

Comment:
Lets move this import to top of the file. 
So all imports will be at one place.
Also, let's change this variable name `pjson`, to something like `PACKAGE_CONFIG` or some similar name